app/processing/invoice-line/get-invoice-line.js

```diff
@@ -0,0 +1,30 @@
+const schema = require('./invoice-line-schema')
+
+const getInvoiceLineByFundingCodeAndPaymentId = require('./get-invoice-line-by-funding-code-and-payment-id')
+
+const getInvoiceLine = async (fundingCode, paymentRequestId) => {
+  const invoiceline = await getInvoiceLineByFundingCodeAndPaymentId(fundingCode, paymentRequestId)
+  const result = schema.validate(invoiceline, {
+    abortEarly: false
+  })
+
+  if (result.error) {
+    throw new Error(`Payment request with paymentRequestId: ${paymentRequestId} does not have the required invoice-line data for funding code ${fundingCode} : ${result.error.message}`)
+  }
+
+  const annualValue = invoiceline.value
+  const quarterlyValue = annualValue > 0 ? annualValue / 4 : 0

```

Comment:
We should wrap this in `Math.trun(...)`.

This is because the values are all in pence, which makes calculations like this much easier to handle rounding issues (which JS is weird at).  Means we don't need the decimals so can be removed.

It's the same approach we took in the payment service.  eg -> https://github.com/DEFRA/ffc-pay-processing/blob/6507b4fbedb23b84c6a1236331ce185d8ae13e46/app/processing/delta/split-to-ledger.js#L43